dd-sdk-android/src/main/kotlin/com/datadog/android/tracing/TracingInterceptor.kt

```diff
@@ -132,9 +131,7 @@ class TracingInterceptor : Interceptor {
         (span as? MutableSpan)?.isError = true
         span.setTag(DDTags.ERROR_MSG, error.message)
         span.setTag(DDTags.ERROR_TYPE, error.javaClass.name)
-        val sw = StringWriter()
-        error.printStackTrace(PrintWriter(sw))
-        span.setTag(DDTags.ERROR_STACK, sw.toString())
+        span.setTag(DDTags.ERROR_STACK, error.loggableStackTrace())
```

Comment:
`ERROR_STACK` is defined as `"error.stack"`:

https://github.com/DataDog/dd-sdk-android/blob/286f23ddf78586ed4b9886ebe6132e8d610740c4/dd-sdk-android/src/main/java/datadog/trace/api/DDTags.java#L18

but the [OT Log fields table](https://github.com/opentracing/specification/blob/master/semantic_conventions.md#log-fields-table) defines it as `"stack"`. I think this isn't correct, no? 🤔

Reply:
Also, the `STACK` field [is defined](https://github.com/opentracing/opentracing-java/blob/a8557d7c7a56210cbcf6bc518fe24b32923aa66f/opentracing-api/src/main/java/io/opentracing/log/Fields.java#L54) in `opentracing-java`, maybe we should use that one instead?

```java
public static final String STACK = "stack";
```

Reply:
Well this is because we read OpenTracing fields and convert them to Datadog fields. 
the DDTags are tags that are specific to datadog, while Fields in the OpenTracing are fields users might attach to span, which we convert to DDTags fields.

Reply:
Ah, so we convert from OT to DD here 👌 I see.